internal/provider/vrf/tier0_vrf_datasource.go

```diff
@@ -37,7 +37,7 @@ type tier0VrfDataSourceModel struct {
 
 type segmentModel struct {
 	Service types.String `tfsdk:"service"`
-	VlanID  types.String `tfsdk:"vlan_id"`
+	VLANID  types.String `tfsdk:"vlan_id"`
```

Comment:
Need another issue for vrf resource and VLANID

Reply:
It's linked to the golangci-lint config, if I don't change this here the lint step fall.